canonical-web-and-design/cn.ubuntu.com
# Merge replace-page with create

`export-page` should either create or replace as necessary. No need for two functions.
## QA

``` bash
make run
```

Go to 127.0.0.1:8010/admin and create a new page.

Create export and migration files:

``` bash
make export-page  # Enter page URL when prompted
git status        # See that it's create a .json and a migration file
```

Now modify some page data in http://127.0.0.1:8010/admin and save the modifications, and export it again:

``` bash
make export-page  # Enter page URL when prompted
git status        # The .json file should have been updated with latest data
                  # And you should see a second migration file to replace the page
```

Now run the site again from scratch:

``` bash
make clean-all run
```

Visit http://127.0.0.1:8010/admin and check your new page with the latest data has been recreated properly.
